### DISCUSSION TITLE
Is it possible to generate models always in singular names with respect to plural names of tables in DB?

### DISCUSSION THREAD

**Discussion by grv07** [SeaQL/sea-orm#1039] [Q&A] [answered]
We have plural names in our db and `sea-orm-cli` also generates our auto-generated Model in the plural. Can we configure the `sea-orm-cli` tool to generate a singular name for our auto generated `model`

**Comment by billy1624** [comment, marked as answer]
Hey @grv07, welcome! Glad you asked :)
Actually, we're working on a PR exactly for this purpose. Please check https://github.com/SeaQL/sea-orm/pull/680

**Reply by grv07** [reply]
Thanks, @billy1624, I will follow the PR and hope it will merge soon :)

**Reply by grv07** [reply]
Hey, @billy1624  Do we have any other PR for this in the process?
 #680 is not looking active anymore.

**Reply by billy1624** [reply]
Feel free to submit another PR. I'd be more than happy to review it :D

**Reply by rmnilin** [reply]
Hello, @billy1624, could you please clarify the need for another PR? Is there an issue with PR #680?
Based on my understanding, the progress of the PR seems to have halted after the author's explanation about creating three different methods for clarity purposes.
Although I may be mistaken, it seems that the PR is now ready for merging.

Also, is there any workaround until the PR is merged?
Apart from manual editing, of course 🤔

**Reply by tyt2y3** [reply]
Unfortunately the PR seems to have diverged from master since then. And there has been a lot of refactoring on master. So work is required to migrate the implementation on top of master, and addressing the comment on coding style.
